src/components/layouts/ConfirmUI.tsx

```diff
@@ -0,0 +1,64 @@
+'use client'
+
+import { StyledBox } from '@/app/detail/ui/styledComponent'
+import { AppMargin, SizeofAppMargin } from '@/hoc/AppMargin'
+import { Box, Stack, Typography, styled } from '@mui/material'
+import { SecondaryBtn } from '../buttons/SecondaryBtn'
+import { PrimaryBtn } from '../buttons/PrimaryBtn'
+
+interface Prop {
+  handleCancel: () => void
```

Comment:
```suggestion
interface ConfirmUIProps {
  handleCancel: () => void
```

It's good convention to name these Component name + "Props"